velox/buffer/Buffer.h

```diff
@@ -544,6 +585,12 @@ class AlignedBuffer : public Buffer {
     }
   }
 
+  void setPool(velox::memory::MemoryPool* pool) {
```

Comment:
this should be a protected method at least?
Consider rename to transferPool()? or keep as the name with comment say Used by memory pool transfer?